openml/flows/__init__.py

```diff
@@ -3,5 +3,5 @@
 from .sklearn_converter import sklearn_to_flow, flow_to_sklearn, _check_n_jobs
 from .functions import get_flow, list_flows, flow_exists, assert_flows_equal
 
-__all__ = ['OpenMLFlow', 'create_flow_from_model', 'get_flow', 'list_flows',
-           'sklearn_to_flow', 'flow_to_sklearn', 'flow_exists']
+__all__ = ['OpenMLFlow', 'get_flow', 'list_flows', 'sklearn_to_flow',
+           'flow_to_sklearn', 'flow_exists', '_check_n_jobs']
```

Comment:
Why do you want to export the private _check_n_jobs function?

Reply:
Weird, I vaguely remember me fiddling around with this, but it seems so unlogical now ..